### DISCUSSION TITLE
E7250 High sierra bootpack (EFI folder)

### DISCUSSION THREAD

**Discussion by nickogaidarov** [SkyrilHD/Dell-E7250-Hackintosh#41] [Q&A]
Hello, SkyrilHD and thank you for your great work! I have tried your EFI pack for Big Sur and everything went smoothly. However it seems that i will need High Sierra instead of Big Sur because of old software i am planning to use. Can you please share an EFI folder for High Sierra? Thank you in advance and keep up the great work!

**Comment by SkyrilHD** [comment]
This EFI should work without an issue: https://github.com/SkyrilHD/Dell-E7250-Hackintosh/releases/tag/v2.8
Only thing for WiFi you need to replace AirportItlwm as 2.8 is for Big Sur. Get the High Sierra kext from [here](https://github.com/OpenIntelWireless/itlwm/releases) and replace it.

**Reply by nickogaidarov** [reply]
Hey! Thank you for your quick reply! I have just tried it as you said and it boots successfuly but after i select EFI (dmg) option it shows the apple logo(without a progress bar) which stays for like 3-4 minutes and after that the computer reboots. :( What could be wrong? Thank you again!

**Reply by SkyrilHD** [reply]
Can you add '-v' to boot-args and send a picture where it hangs?

**Reply by nickogaidarov** [reply]
> Can you add '-v' to boot-args and send a picture where it hangs?

I guess i am using a release package and i can't find where to put -v as an argument.(Sorry i am noob). I used the same 4GB usb flash driver properly formatted to FAT32 with which i succesfuly managed to install Big Sur. Today i tried Mojave as well, however it still got stucked on the apple logo without a progress bar followed by a reboot. I also tried with 16GB USB 3.0 flash drive with no luck. Thank you again for your help and patience!

**Reply by nickogaidarov** [reply]
I found the boot args in config.plist. So i have to put -v just after it?

**Reply by nickogaidarov** [reply]
I am not sure if i've done it correctly but i put "-v" under boot-args between strings. It booted and the only text i got was **End SetConsoleMode**

**Reply by nickogaidarov** [reply]
Here is a link of my EFI folder: https://mega.nz/file/IPJWHBZD#J8jWa44KF6kpOjZuSF5F_Z3N9stWg_KvzNDrXnvNUPI

**Reply by nickogaidarov** [reply]
Tried Big Sur again and it boots without an issue. Could it be different settings in the BIOS for High Sierra?

**Reply by SkyrilHD** [reply]
Could you try disabling "Enable Legacy Option ROMs" in the BIOS?

**Reply by nickogaidarov** [reply]
Still the same with it disabled. Thank you!

**Reply by SkyrilHD** [reply]
I just booted the High Sierra Installer myself and it works without an issue.

I downloaded (as suggested) the v2.8 EFI and the only change I did was the '-v' I added in the config. No other changes.

I was also stuck at End SetConsoleMode but after 5 seconds it booted right up. My guess is your installer is broken.

![IMG_20211024_190429](https://user-images.githubusercontent.com/28839925/138604801-da53f8f9-4276-412a-90ec-ccc1d045fb37.jpg)

**Reply by nickogaidarov** [reply]
Thank you again for your time and effort to help me! I really appreciate it! I tried downloading it twice(with the two commands in the guide provided) before i post here. I will try with another USB drive and i will play with the BIOS settings. 

P.S.: Can you share your installer and EFI folder just in case. Can i donate to the project?

**Reply by SkyrilHD** [reply]
Here you go, this is the EFI I tested: [EFI.zip](https://github.com/SkyrilHD/Dell-E7250-Hackintosh/files/7405990/EFI.zip) (had to remove AirportItlwm, otherwise I cannot upload)
The installer is quite big (offline installer) and would take days to upload with my internet speed. Are you using an online or offline installer?
I don't accept donations for various reasons.

EDIT: My BIOS settings are [here](https://github.com/SkyrilHD/Dell-Exx50-Hackintosh#recommended-bios-settings)

**Reply by nickogaidarov** [reply]
I will try tomorrow with your EFI and your BIOS settings. I downloaded the installer with macrecovery.py as i tried both of them:

# High Sierra(10.13)
python macrecovery.py -b Mac-7BA5B2D9E42DDD94 -m 00000000000J80300 download
python macrecovery.py -b Mac-BE088AF8C5EB4FA2 -m 00000000000J80300 download

Thank you again!

**Reply by SkyrilHD** [reply]
I suggest you doing this way: Install any macOS (i.e. Big Sur) and download the full installer and create a bootable USB. It is time consuming but I only had better luck with offline installers.
